panel/auth.py

```diff
@@ -106,6 +110,8 @@ class OAuthLoginHandler(tornado.web.RequestHandler):
 
     _error_template = ERROR_TEMPLATE
 
+    _login_endpoint = '/login'
```

Comment:
You are hard coding the login endpoint here. if we use:
```
pn.serve(..., login_endpoint="/auth")
```
This will result a redirect to `/login` with a 404 error:
```
404 GET /login (::1) 1.10ms
```